packages/integrations/cloudflare/src/index.ts

```diff
@@ -39,6 +39,8 @@ const SHIM = `globalThis.process = {
 
 const SERVER_BUILD_FOLDER = '/$server_build/';
 
+const matchNoneIncludePattern = '/MATCH_NONE_INCLUDE_PATTERN';
```

Comment:
It would be nice to add some comment here. Ideally, a brief explanation or a link to the documentation that explains this magic value.